scripts/generate-cluster-creds.sh

```diff
@@ -40,51 +40,56 @@ main() {
   SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 
   if [ "$CLUSTER_TYPE" == "minikube" ]; then
-    cat > cluster-credential.json <<EOF
+cat <<EOF | yq e -P - > k8s_config.yaml
 {
-  "name": "$CLUSTER_NAME",
-  "master_ip": "https://kubernetes.default.svc:443",
-  "certs": "$(cat ~/.minikube/ca.crt | awk '{printf "%s\\n", $0}')",
-  "client_certificate": "$(cat ~/.minikube/profiles/minikube/client.crt | awk '{printf "%s\\n", $0}'))",
-  "client_key": "$(cat ~/.minikube/profiles/minikube/client.key | awk '{printf "%s\\n", $0}'))"
+  "k8s_config": {
+    "name": "dev",
```

Comment:
Should it be `$CLUSTER_NAME` instead of `dev`?

Reply:
Yup, thanks for spotting this!